doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -300,6 +300,51 @@ can be launched explicitly using :func:`create_launcher() <ansys.fluent.core.lau
   >>> pim_solver_session = pim_solver_launcher()
 
 
+Launching a Fluent session in container mode using Docker Compose and Podman Compose
+------------------------------------------------------------------------------------
+
+You can launch an Ansys Fluent session in container mode using PyFluent along with either Docker Compose or Podman Compose. 
+This allows you to manage multi-container workflows with a consistent and reproducible simulation environment.
+
```

Comment:
We can cut out the bits of information that the redaer already knows, leaving something like this:

**Launching Fluent in Container Mode with Docker or Podman Compose**
Use PyFluent with Docker Compose or Podman Compose to run Fluent in a consistent, reproducible containerized environment.

Reply:
Done. Thank you.